packages/manager/apps/pci-instances/src/types/image/api.types.ts

```diff
@@ -0,0 +1,16 @@
+import { DeepReadonly } from '../utils.type';
+
+export type TImageDto = DeepReadonly<{
```

Comment:
This DTO seems weird :
planCode is nullable on API
visibility shouldn't be a string
status should be an enum or be split into usable booleans